src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -20,12 +24,16 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and who stays in the specified block"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose characteristics match the"
+            + " given keywords and parameters"
             + " and displays them as a list with index numbers.\n"
             + "Parameters: "
-            + "[" + PREFIX_NAME + "KEYWORDS [MORE KEYWORDS] " + "]"
-            + "[" + PREFIX_BLOCK + "BLOCK " + "]"
+            + "[" + PREFIX_NAME + "KEYWORDS [MORE KEYWORDS]" + "] "
+            + "[" + PREFIX_BLOCK + "BLOCK" + "] "
+            + "[" + PREFIX_FLOOR + "FLOOR" + "]"
+            + "[" + PREFIX_ROOM_NUMBER + "ROOM_NUMBER" + "] "
+            + "[" + PREFIX_MATRICULATION_NUMBER + "MATRICULATION_NUMBER" + "] "
+            + "[" + PREFIX_GENDER + "GENDER" + "] "
             + "[" + PREFIX_STUDENT_GROUP + "STUDENT_GROUP " + "]"
```

Comment:
Sorry this isn't exactly your issue, but actually should it be `"[" + PREFIX_STUDENT_GROUP + "STUDENT_GROUP " + "]..."` instead? Reason being the user can actually do `find s/dance s/basketball`.

Reply:
isn't it already that?

Reply:
Okay never mind, my bad.